py/visdom/server/__main__.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
Learn something new every day. I didn't know you could run folders via a `__main__` file.

Reply:
Same for me, didn't know this before as well. :)